### PR TITLE
Patch/vinyl

### PR DESCRIPTION
**Pull request by warloque**
Patched with Vinyl instead of now deprecated gulp-util.
Please, update the code base and the dependencies badge)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +31,7 @@
     "test": "mocha"
   },
   "engines": {
-    "node": ">= 0.10"
+    "node": ">= 10.6"
```
Comment (jbdemonte) [inline package.json]: could you update it to the LTS at least

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "gulp-file-insert",
   "description": "Replace custom tokens by files content",
-  "version": "1.0.3",
+  "version": "1.0.4",
```
Comment (jbdemonte) [inline package.json]: update to version 2 plz

**Comment by jbdemonte** [comment]
Could you fix the CI plz

File: package.json (hunk 1 above)
Comment (warloque) [reply package.json]: I updated to >=10.6 as it's LTS at the moment. I've found no 10.6LTS-syntax available...

As the LTS concept is one that's controlled by the Node.js LTS Working Group, which is a separate team from the npm CLI, this is likely to be tricky to get into npm. The CLI team isn't going to have time to get to this in the near-to-medium term.

**Comment by warloque** [comment]
UPD: updated version, dependencies and travis tests, both have passed OK.

File: package.json (hunk 1 above)
Comment (jbdemonte) [reply package.json]: LTS for version 6 or 8 should be more flexible for users
https://github.com/nodejs/Release#release-schedule

**Comment by warloque** [comment]
UPD: done rollback and CI test passed on node LTS 8.11.3

**Comment by warloque** [comment]
Is everything OK? If so, waiting for you accepting the patch)

**Pull request merged**